ray-kubernetes/head.yml

```diff
@@ -0,0 +1,44 @@
+#replace the image if you built your own from a modified Dockerfile
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: ray-head
+spec:
+  type: "LoadBalancer"
+  ports:
+  - name: redis
+    port: 6379
+    targetPort: 6379
+  - name: plasma
+    port: 12345
+    targetPort: 12345
+  selector:
+    app: ray-head
+---
+apiVersion: v1
+kind: Pod
```

Comment:
Just curious, why is ray head a Pod instead of a Deployment? If we make it into a deployment it can be managed under replication controller.

Reply:
Why do we need replication control for head?